README.md

```diff
@@ -666,6 +663,122 @@ const updateContractTransaction: AccountTransaction = {
 
 Finally, to actually update the contract on the chain, send the constructed `updateContractTransaction` to the chain using `sendAccountTransaction`. (See [Send Account Transaction](#Send-Account-Transaction) for how to do this)
 
+## Smart contract with parameters
+In the previous sections we have seen how to initialize and update contracts without parameters. In this section we will describe how to initialize and update contracts with parameters.
+The user should provide the input in the JSON format specified in [our documentation](https://developer.concordium.software/en/mainnet/smart-contracts/references/schema-json.html).
+
+Let us consider the following example where the contract's initialization parameters are the following structure:
```

Comment:
@shjortConcordium  we didn't added any of these smart contract regarding the tests. shall we add those into resources folder under test?

Reply:
Yes, please, just the source files, I think. It is just that I don't want to refer to/rely on anything private, like if we want to move these test off of Testnet.

Reply:
Did you not have all the contracts anymore, it seems that there are more parameter tests than contracts uploaded, also the names don't match, so it's hard to get an overview.

Reply:
@shjortConcordium we have added all the smart contract there may be more tests than the number of smart contracts because we used same contracts for init and multiple receive functions(update contract). I will update the smart contract folder along with the contract names.

Reply:
Will it makes scene if I rename the current each smartcontract folder as smartcontract-{contractName} ?

Reply:
They are already in a folder called smartcontracts, but I think it would make sense to call them {contractName}.

Reply:
@shjortConcordium added please check and let me know.